build-system/exec.js

```diff
@@ -46,6 +59,16 @@ exports.exec = function(cmd) {
 }
 
 /**
+ * Starts off the provided command in a background process.
+ *
+ * @param {string} cmd Command line to execute.
+ */
+exports.execAsync = function(cmd) {
+  let p = spawnProcessAsync(cmd, {'stdio': 'inherit', 'detached': 'true'});
```

Comment:
You read my mind :) I'm writing a new gulp task that starts the background server and returns after the server has started.